packages/mask/src/plugins/MaskBox/SNSAdaptor/components/DetailsTab.tsx

```diff
@@ -6,7 +6,7 @@ const useStyles = makeStyles()((theme) => ({
     main: {
         height: 360,
         overflow: 'auto',
-        marginBottom: theme.spacing(2.75),
+        padding: 16,
```

Comment:
use `theme.spacing` to ensure consistency of spacing.
```suggestion
        padding: theme.spacing(2),
```